content/news/fpc-24-registration.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Freshmen Programming Contest 2024"
+date: 2024-03-13T12:00:00+02:00
+tags: ["FPC 2024"]
+draft: true
+---
+Are you a freshmen and do you want to solve puzzles in a team with your friends?
+Then the Freshmen Programming Contest is for you!
+
+On the 4th of May, the annual Freshmen Programming Contest will be held.
+The puzzles are suited for both beginners and experts, so don't be afraid to join!
+This contest is a great opportunity for you to sharpen your programming and puzzling skills!
+
+The event lasts approximately from 10:30 until 17:30. More details will be sent
+when you register. Teams with non-freshmen members are able to join but will be
+assigned to the spectator pool, meaning they do not count towards the final ranking.
+
+After getting your ticket [here](https://wisv.ch/fpc), you need to fill in the [registration form](https://wisv.ch/fpcregistration)
+to complete your registration. All team members need to both get a ticket and fill in the registration form!
```

Comment:
> All team members need to both get a ticket and fill in the registration form!

I have re-structued the form so that all team members fill it in individually. We can discuss this in the next meeting, but in my head that made a bit more sense.

Reply:
Is this smart? not all members are gonna use the same name or more then 3 people can register under the same team name. I also thought that the ticket system had the feature now that it can just link to our form, so the team members don't have to do two actions(fill in the form and buy a ticket).

Also tomorow is the evaluation meeting with the board, so we most likely don't have the time discuss this.

Reply:
I agree with Verwoerd here: the CH tickets should(?) not be necessary, and for consistency it's probably best if the entire team registers in one go. If they want to change the members of their team, they can always send an email :slightly_smiling_face: